### ISSUE TITLE
Comments for uploaded files are written & displayed incorrectly via descript.ion file

### ISSUE THREAD

**Issue by jbrown123** [rejetto/hfs#661]
### BEFORE

- Are you using a reverse-proxy, like nginx?

Not using one, also not applicable

### To Reproduce
Upload a file via HFS to an upload enabled folder
Click the comment button and put in more than one word separated by spaces
Upload the file

### Describe the bug
The upload process injects &#0032; for each space or other non-alphanumeric character

### Expected behavior
I expected the comment to be recorded as sent. At the very least, I expect the output to interpret the HTML escaped characters corretly.

### Screenshots
![image](https://github.com/rejetto/hfs/assets/636969/e43f9948-f87b-4d61-8ea2-3418cbda1c48)

### Environment (please complete the following information):
 - OS: Windows 10, 64 bit
 - HFS Version 0.52.9 (latest stable)
 - Browser chrome (doesn't matter)

### Additional context
None

**Comment by rejetto** [comment]
wow!
thanks for reporting.
this bug doesn't apply to the set-comment made from the list

**Comment by jbrown123** [comment]
There's a way to set comments elsewhere? Other than editing the descript.ion file, where can they be set?

**Comment by jbrown123** [comment]
Commenting from a phone is painful

**Comment by rejetto** [comment]
If you can upload, you should also see the "comment" command when you open
the file menu, from the files list

**Comment by jbrown123** [comment]
Ah, yes. I see that. And I can confirm that it works mostly correctly.

It doesn't support a leading return in the comment (it gets stripped). You can manually edit the descript.ion file and add a leading return and that does work correctly. (I prefer the v2 style comments where they appear below the item rather than starting to the right of it.)

Any chance we can get rid of the quotes around the comment?

Maybe make both of these options (comments beside / comments below and with or without quotes)?

**Comment by rejetto** [comment]
you can customize the style of the comments of course, and make all the changes you said.
it's CSS, so it heavilty depends on your knowledge on the topic.
once you devise all the changes that you need, you can enter your new style rules like described at https://github.com/rejetto/hfs/wiki/Customization

to know what rules you need you should study how the page is done, as there's no documentation about it yet.
People making web pages know how to do that even without documentation: you right click on the comment, click inspect.
You'll see that you are on a div with class entry-comment. This "studying" tool is nothing specific to hfs, but all web pages, so I expect some guide/documentation to exist somewhere.

> Maybe make both of these options (comments beside / comments below and with or without quotes)?

the possible styles are virtually infinite, the option is not the right way to go.
Someone can make a simple plugin/theme that just change the style of the comments, and you can install it.

**Comment by rejetto** [comment]
for example, this can be a good start
```css
.entry-comment { display: block }
.entry-comment::before, .entry-comment::after { display: none }
```

**Comment by rejetto** [comment]
this is was fixed in 0.53.0-rc18
